iso15118/secc/secc_settings.py

```diff
@@ -38,6 +38,7 @@ class Config:
         "DIN_SPEC_70121",
         "ISO_15118_2",
         "ISO_15118_20_AC",
+        "ISO_15118_20_DC",
```

Comment:
This could be removed as it is not fully supported.

Reply:
what if we keep it and add a note that DC is under development yet?

Reply:
pls check ;)